fs-sim/src/System/FS/Sim/Error.hs

```diff
@@ -66,7 +67,8 @@ import           System.FS.API.Types
 
 import           System.FS.Sim.MockFS (HandleMock, MockFS)
 import qualified System.FS.Sim.STM as Sim
-import           System.FS.Sim.Stream
+import           System.FS.Sim.Stream (Stream)
+import qualified System.FS.Sim.Stream as Stream
```

Comment:
`System.FS.Sim.Stream` is re-exported by the `Error` module, and qualified imports are not re-exported

Reply:
No longer relevant